commit df92917582d785255cbf4b3bb13c24c7d1609993 (refs/changes/51/4151/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-06-09T04:04:00-07:00 (1 year, 4 months ago)
    
    MANATEE-398 manatee-adm history -j doesn't understand v1 timestamps
    Reviewed by: Richard Bradley <richard.bradley@joyent.com>
    Approved by: Richard Bradley <richard.bradley@joyent.com>

diff --git a/bin/manatee-adm b/bin/manatee-adm
index 7e34493..13dbd22 100755
--- a/bin/manatee-adm
+++ b/bin/manatee-adm
@@ -719,7 +719,7 @@ ManateeAdm.prototype.do_history = function (subcmd, opts, args, cb) {
 
             if (evt.action) {
                 return ({
-                    'TIME': evt.date.toISOString(),
+                    'TIME': evt.time.toISOString(),
                     'G#': '-',
                     'MODE': '-',
                     'FRZ': '-',
